### ISSUE TITLE
truecrypt-3.0.zip (Windows binaries only, no source) possibly located

### ISSUE THREAD

**Issue by qwerty12** [DrWhax/truecrypt-archive#8]
Hi,

tl;dr part here:
I think I've found truecrypt-3.0.zip [here](http://www.dslreports.com/forum/r12085073-Truecrypt-3.0) with the file name `truecrypt3.0.zip`. I have downloaded the same file on two computers connected to the same network. Between the two computers the checksums match:
MD5: `2EEA7841E7EACB161F69EAD8F7793CE7`
SHA-256: `BFC2D6875DE92EBB80577BCDAF7017BB33F371723C82AC9D161E9568CFEDBBE6`

_However_, searching for the checksums (MD5, SHA1, SHA-256) doesn't reveal anything on the Internet.

(On computer 1 I used HashTab and on computer 2, I used sums.zip from [here](http://blog.nfllab.com/archives/152-Win32-native-md5sum,-sha1sum,-sha256sum-etc..html) - the non-updated version)

---

Using GPG4Win 2.2.1 (the digital signature on the EXE along with the SHA1 matched what was on the GPG4Win site) on computer 2 (which has a relatively fresh install of Windows 8.1 [no updates installed] and Asus, Intel, ASMedia and NVIDIA drivers along with Firefox), I was able to follow the instructions [here](https://madiba.encs.concordia.ca/~x_decarn/truecrypt-binaries-analysis/) to successfully verify the ZIP file in Kleopatra against the `TrueCrypt-Foundation-Public-Key.asc` and the `truecrypt-3.0.zip.sig` files present in this very repository. The `TrueCrypt-Foundation-Public-Key.asc` file present on my computer had the same SHA1 sum as what is present on the Concordia site. The fingerprint of the asc matched what I was seeing on various Internet sources.

For ~~redundancy~~ good measure, I also followed the instructions here on verification through the Command Prompt (using. again. the `gpg.exe` that is part of GPG4Win) and received a "good signature from TrueCrypt Foundation" message.

The date returned when verifying the ZIP's sounds plausible - the file is supposedly signed on 2004-12-10 and the file over at DSLReports was uploaded at 2004-12-11. 

There is no digital signature present on the actual executables or driver but I'm guessing that that's intentional as the files in the 3.0a ZIP from here are also lacking one.

Sorry for the length of this message but I followed your advice and secured a tin-foil hat to my head. I also apologise for not being able to submit a pull request, but I've already downloaded this repository a few times and I don't want to fork such a large repository just to add one small file.

**Comment by stefansundin** [comment]
Nice detective work! I appreciate that you secured a tin-foil hat. :)

```
$ gpg --verify truecrypt-3.0.zip.sig
gpg: Signature made Fri Dec 10 06:59:46 2004 PST using DSA key ID F0D6B1E0
gpg: Good signature from "TrueCrypt Foundation <contact@truecrypt.org>"
```

Thank you!!
